res/PyTorchExamples/README.md

```diff
@@ -0,0 +1,41 @@
+# PyTorch examples
+
+Python example to convert PyTorch to ONNX/TF/tflite models
+
+## Package versions
+
+- Python 3.X
+- torch==1.7.0
+- onnx=1.7.0
```

Comment:
```suggestion
- onnx==1.7.0
```

Reply:
And, FYI, `pytorch` 1.7.0 can't be installed using pip in python 3.5.2. When I tried to install it in 3.6.9, it works.